rbc/omnisci_array.py

```diff
@@ -29,6 +30,56 @@ def key(self):
         return self.dtype
 
 
+class Array(object):
+    pass
+
+
+# XXX there should be a better way to find element size
+bytes_map = {
+    'int8': 1,
+    'int16': 2,
+    'int32': 4,
+    'int64': 8
+}
```

Comment:
This is "necessary" for the final bitcast instruction. There's probably a better way to detect the type length using the typesystem.